README says to use "service_name=xxx" but Makefile expects "service=xxx"

example:

Error:
```
make create_job service_name=cloudfront
'service' variable must be defined
make: *** [require_service] Error 1
```

Success:
```
make create_job service=cloudfront
test -n CloudFront_Partitioner || (echo "cloudfront service not found" && echo 1)
aws glue create-job --name CloudFront_Partitioner_LogMaster_v5.3.0 \
		--description "CloudFront_Partitioner Log infra generator" \
		--role AWSGlueServiceRoleDefault \
...
...
```

```
$ make --version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
```

--- a/README.md
+++ b/README.md
@@ -73,7 +73,7 @@ To deploy the scripts to your own bucket, override the `RELEASE_BUCKET` environm
 
 ```shell
 RELEASE_BUCKET=<YOUR_BUCKET_NAME> make private_release
-RELEASE_BUCKET=<YOUR_BUCKET_NAME> make create_job service_name=<SERVICE_NAME>
+RELEASE_BUCKET=<YOUR_BUCKET_NAME> make create_job service=<SERVICE_NAME>
 ```
 
 `SERVICE_NAME` is the key in the config file. Don't change those keys as they are also references to the actual Glue scripts.

